tests/unit/plugins/service/test_event_consumer.py

```diff
@@ -363,12 +369,17 @@ def mock_create_gevent_msg(consumer, mocker, monkeypatch):
 
 @pytest.mark.asyncio
 async def test_handle_pubsub_msg_old(mocker, monkeypatch, consumer,
-                                     raw_msg_data, creation_audit_log_data,
+                                     publish_time, raw_msg_data,
+                                     creation_audit_log_data,
                                      caplog, pubsub_msg, mock_get_and_validate,
                                      mock_create_gevent_msg):
     """Too-old message."""
-    mocker.patch(DATETIME_PATCH, conftest.MockDatetime)
-    pubsub_msg.publish_time -= datetime.timedelta(
+
+    def now(tz):
+        return publish_time
```

Comment:
FWIW I think this could be done in a `lambda` directly in the `mocker.patch`

Reply:
Oh, good call.  Kept evolving.  I knew there was a way to make it even shorter. :)